test/compilable/extra-files/header1.d

```diff
@@ -10,6 +10,8 @@ static assert(true, "message");
 
 alias double mydbl;
 
+alias fl = function () in {} body {};
```

Comment:
How about adding a function that actually uses `do` to the testsuite as well? (Consistency ...)

Reply:
Now fixed.